pdns/dnspacket.hh

```diff
@@ -167,6 +173,9 @@ private:
   string d_rawpacket; // this is where everything lives 4
   int d_maxreplylen;
   string d_ednsping;
+  uint8_t d_ednsversion;
+  uint8_t d_ednsrcode;
```

Comment:
EDNS rcode could theoretically be 12 bits, maybe note this in a comment to reduce surprises for whomever has to deal with that 30 years from now?